GeeksCoreLibrary/GeeksCoreLibrary.csproj

```diff
@@ -74,6 +74,7 @@
     <PackageReference Include="System.Threading" Version="4.3.0" />
     <PackageReference Include="Twilio" Version="6.2.5" />
     <PackageReference Include="WebMarkupMin.AspNetCore3" Version="2.13.8" />
+    <PackageReference Include="ZstdNet" Version="1.4.5" />
```

Comment:
I looked through the code, but it doesn't look like ZstdNet isn't used anywhere. Was this package added by accident?

Reply:
The Mysql X Dev API makes use of this library.